src/renderer/components/+preferences/kubectl-binaries.tsx

```diff
@@ -29,8 +29,8 @@ export const KubectlBinaries = observer(({ preferences }: { preferences: UserPre
         <FormSwitch
           control={
             <Switcher
-              checked={preferences.downloadKubectlBinaries}
-              onChange={v => preferences.downloadKubectlBinaries = v.target.checked}
+              checked={UserStore.getInstance().downloadKubectlBinaries}
```

Comment:
Maybe better to extract to `const` within current  function scope and re-use below?

Reply:
Sounds good